exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/using_a_script_languages_container.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<p>Can you show an example of a Dockerfile and mark the sections that can be changed "with care" and carelessly? And yes, I don't like the expression "with care" because it doesn't explain what kind of care I need to apply - what I need to be mindful of.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>The previous commands prints a link to the Dockerfile, which the user can click: The Dockerfile will be opened in a new tab in the Jupyter IDE. I can show you in a phone call.</p>